### PR TITLE
feat(JobCard): Replace companyLogoUrl to company.logoUrl

### PR DESCRIPTION
**Pull request by ZhenYongSeekAsia**
** Please provide as much detail as possible, but feel free to remove irrelevant sections **

change to adapt search schema

BREAKING CHANGE:

{{ Description of breaking API change }}

RFC URL:

{{ URL of associated RFC for API change }}

MIGRATION GUIDE:

Before:

```js
{{ Old code }}
```

After:

```js
{{ New code }}
```

EXAMPLE USAGE:

```js
{{ Code }}
```

### CODE REVIEW

File: react/JobCard/JobCard.js [hunk 1]
```diff
@@ -223,6 +224,8 @@ export default class JobCard extends React.Component {
 
     const { shelfSectionOpen } = this.state;
 
+    const { logoUrl } = _get(job, 'company');
```
Comment (kosanna) [inline react/JobCard/JobCard.js]: you need to default to {} here _get(job, 'company', {})

File: react/JobCard/JobCard.test.js [hunk 2]
```diff
@@ -7,7 +7,8 @@ import omit from 'lodash/omit';
 const defaultJob = {
   company: {
     name: 'SEEK Asia',
-    link: '/jobCard'
+    link: '/jobCard',
+    LogoUrl: 'https://siva.jsstatic.com/my/94463/images/logo/94463_logo_0_48885.png'
```
Comment (kosanna) [inline react/JobCard/JobCard.test.js]: isn't it supposed to be lowercase?

**Comment by kosanna** [comment]
Some more demo updates like [here](https://github.com/seekinternational/seek-asia-style-guide/blob/3e41216a31473dd762cbd7126c6ea25495802970/react/JobCard/JobCard.demo.js#L403) and it's good to go

File: react/JobCard/JobCard.test.js (hunk 2 above)
Comment (ZhenYongSeekAsia) [reply react/JobCard/JobCard.test.js]: yes, it suppose lowercase

**Pull request merged**